scripts/shell/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pc-ble-driver-py>=0.15.0
+nrfutil>=6.0.0
```

Comment:
i've tested only on v.6.1.0 so can't be sure if this works from my perspective
if scripts works with v6.0.0 ignore this comment :)